doc/monitoring/getting_started.rst

```diff
@@ -1,68 +1,140 @@
 ..  _monitoring-getting_started:
 
-Monitoring: getting started
-===========================
+Getting started with monitoring
+===============================
 
-.. _monitoring-getting_started-tarantool:
+If you use Tarantool version below `2.11.1 <https://github.com/tarantool/tarantool/releases/tag/2.11.1>`__,
+it is necessary to install the latest version of ``metrics`` first. For details,
+see :ref:`Installing the metrics module <install>`.
 
-Tarantool
----------
+..  _monitoring-getting_started-usage:
 
-First, install the ``metrics`` package:
+Using the metrics module
+------------------------
 
-..  code-block:: console
+..  note::
 
-    $ cd ${PROJECT_ROOT}
-    $ tarantoolctl rocks install metrics
+    The module is also used in applications based on the Cartridge framework. For details,
+    see the :ref:`Getting started with Cartridge application <metrics-getting_started_cartridge>` section.
 
-Next, require it in your code:
+#.  First, set the instance name and start to collect the standard set of metrics.
+    Also, you can set a global label for your instance.
 
-..  code-block:: lua
+    ..  code-block:: lua
 
-    local metrics = require('metrics')
+        metrics.cfg(labels = {alias = 'my-instance'})
```

Comment:
```suggestion
        metrics.cfg{labels = {alias = 'my-instance'}}
```
(Otherwise, it's an invalid expression in terms of syntax.)

Reply:
Fixed